tests/unit_tests/test_wave_06.py

```diff
@@ -106,20 +104,17 @@ def test_swap_best_by_category():
     # Act
     result = tai.swap_best_by_category(
         other_vendor=jesse,
-        my_priority="Clothing",
-        their_priority="Decor"
+        my_priority="Clothing", # Tai gets item f --> a, b, f
+        their_priority="Decor" # Jesse gets item c --> d, e, c
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That the results is truthy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories, including the items which were swapped from one vendor to the other
+    assert result
```

Comment:
This checks that `result` is any truthy value (a list with elements inside it, a non-empty string, a dictionary with k/v pairs would all pass this assertion). We should be a little more explicit and check `assert == True`